packages/ember-views/lib/index.js

```diff
@@ -37,6 +39,8 @@ const ViewUtils = Ember.ViewUtils = {};
 ViewUtils.isSimpleClick = isSimpleClick;
 ViewUtils.getViewClientRects = getViewClientRects;
 ViewUtils.getViewBoundingClientRect = getViewBoundingClientRect;
+ViewUtils.getRootViews = getRootViews;
+ViewUtils.getChildViews = getChildViews;
```

Comment:
@rwjblue does this seem good to you?

I think `getViewBounds` is in the same ballpark in terms of stability, so if we are adding these, maybe we should add that here too